v3/js/iframe-embed.js

```diff
@@ -63,6 +75,7 @@ $(document).ready(function() {
   var heapPrimitivesBool = (queryStrOptions.heapPrimitives == 'true');
   var textRefsBool = (queryStrOptions.textReferences == 'true');
   var cumModeBool = (queryStrOptions.cumulative == 'true');
+  rawInputLst = queryStrOptions.rawInputLst;
```

Comment:
(as described by PythonTutor people)  This is an "ugly" variable that needs "refactoring" This global variable is used by several modules to handle user input, there is not passing information between functions but instead all of them have access to it. I agree that it need refactoring but I don't know how hard it would be